superset/tasks/schedules.py

```diff
@@ -533,6 +535,9 @@ def schedule_email_report(  # pylint: disable=unused-argument
     name="alerts.run_query",
     bind=True,
     soft_time_limit=config["EMAIL_ASYNC_TIME_LIMIT_SEC"],
+    autoretry_for=(NoSuchColumnError, ResourceClosedError,),
+    retry_kwargs={"max_retries": 5},
```

Comment:
add a todo and the link to the issue - to get rid of retry once the underlying issue is resolved